test/google_crawler/keywords_test.exs

```diff
@@ -98,6 +102,57 @@ defmodule GoogleCrawler.KeywordsTest do
     end
   end
 
+  describe "list_keywords/1 with result count filtered" do
+    test "filters keywords with search results more than the given amount if given > in operation" do
+      user = insert(:user)
+      keyword1 = insert(:keyword, user: user)
+      _keyword2 = insert(:keyword, user: user)
+      insert(:search_result, url: "example1.com", keyword: keyword1)
+      insert(:search_result, url: "example2.net", keyword: keyword1)
+
+      [keyword1_in_db] = Keywords.list_keywords(user.id, %{"result_count_operation" => ">", "result_count_value" => "1"})
```

Comment:
It hards to know where does the `result_count_value` comes from, should we add it into the `insert(:search_result, url: "example1.com", keyword: keyword1)` so it becomes more clear for this test?

```
...
insert(:search_result, url: "example1.com", keyword: keyword1, result_count_value: 2)

[keyword1_in_db] = Keywords.list_keywords(user.id, %{"result_count_operation" => ">", "result_count_value" => "2"})

assert keyword1_in_db.id == keyword1.id
```

Reply:
`result_count_value` is not a column attribute of search result, it's a count of search results that belong to the keyword. 

For this test, it creates 2 search result count for `keyword1` and list the keyword with search result count more than 1 🙌

Reply:
Oh I see, so there will be `multiple search results` per keyword :)

It's ok but next time I would suggest putting the search result into the same keyword record or something like having 1 keyword record with only 1 result record, we could store the result as `array or json` that avoid creating too many DB records as for example like 1 keyword has 10 search results, with 100 keywords we could easily have 10_000 search result records.